src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,398 @@
-{%- endcapture -%}
-{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-%}
+In Python you can either capture a caught exception or the one currently held in `sys.exc_info()` by not passing an argument:
+
+```python
+from sentry_sdk import capture_exception
+
+try:
+    a_potentially_failing_function()
+except Exception as e:
+    # Alternatively the argument can be omitted
+    capture_exception(e)
+```
+
+### Capturing Messages
+
+Another common operation is to capture a bare message. A message is just some textual information that your app can send to Sentry. Typically, messages are not emitted, but there are situations when this is useful.
+
+```python
+from sentry_sdk import capture_message
+
+capture_message('Something went wrong')
+```
+
+## Releases
+
+A release is a version of your code that you deploy to an environment. When you give Sentry information about your releases, you unlock many new features:
+
+- Determine the issue and regressions introduced in a new release
+- Predict which commit caused an issue and who is likely responsible
+- Resolve issues by including the issue number in your commit message
+- Receive email notifications when your code gets deployed
+
+After configuring your SDK, setting up releases is a 2-step process:
+
+1. [Create Release and Associate Commits]({%- link _documentation/workflow/releases.md -%}#create-release)
+2. [Tell Sentry When You Deploy a Release]({%- link _documentation/workflow/releases.md -%}#create-deploy)
+
+For more information, see [Releases Are Better With Commits](https://blog.sentry.io/2017/05/01/release-commits.html).
+
+## Adding Context
+
+You can also set context when manually triggering events.
+
+### Setting Context
+
+Sentry supports additional context with events. Often this context is shared among any issue captured in its lifecycle, and includes the following components:
+
+**Structured Contexts**
+
+: Structured contexts are typically set automatically.
+
+[**User**](#capturing-the-user)
+
+: Information about the current actor
+
+[**Tags**](#tagging-events)
+
+: Key/value pairs which generate breakdown charts and search filters
+
+[**Level**](#setting-the-level)
+
+: An event's severity
+
+[**Fingerprint**](#setting-the-fingerprint)
+
+: A value used for grouping events into issues
+
+[**Unstructured Extra Data**](#extra-context)
+
+: Arbitrary unstructured data which the Sentry SDK stores with an event sample
+
+### Extra Contex
+
+In addition to the structured context that Sentry understands, you can send arbitrary key/value pairs of data which the Sentry SDK will store alongside the event. These are not indexed, and the Sentry SDK uses them to add additional information about what might be happening:
+
+```python
+from sentry_sdk import configure_scope
+
+with configure_scope() as scope:
+    scope.set_extra("character_name", "Mighty Fighter")
+```
+
+### Unsetting Context
```

Comment:
We're telling them how to set extra context, then telling them about unsetting context, then telling them about setting more context. Probably something to be done with order of operations here. This example of unsetting context also doesnt suggest any kind of unset behavior. We should describe how to clear context within each specific context as the APIs may vary.